index.html

```diff
@@ -152,5 +152,16 @@ <h1 id="title">mcMerge</h1>
     <noscript>
       You need Javascript to proceed.
     </noscript>
+
+    <hr />
+
+    <div id="footer">
+      <div id="source-link">
+        <a href="https://github.com/mozilla/mcMerge"> mcMerge Source </a>
+      </div>
+      <div id="mcmerge-revision">
+        <a href="https://mcmerge.paas.allizom.org/deploy.txt"> Deployed revision </a>
```

Comment:
I switched it to a relative link and also added some CSS so it only shows up on the https://mcmerge.paas.allizom.org/ site, hidden everywhere else. 

It's a -moz prefixed rule, so it only hides in Firefox, but I think that'd be okay.

We'd have to update that rule should the hosting move somewhere else.

If you're okay with that, @edmorley I can merge this patch in.